convex/schema.ts

```diff
@@ -0,0 +1,9 @@
+import { defineSchema, defineTable, s } from "convex/schema";
+
+export default defineSchema({
+    employees: defineTable({
+      emp_no: s.number(),
+      first_name: s.string(),
+      last_name: s.string(),
```

Comment:
would camelCase be more idiomatic? do they use snake case in their schema elsewhere?

Reply:
They do in SQL. But I can do camelCase here.